pkg/reconciler/knativeserving/knativeserving.go

```diff
@@ -108,6 +108,7 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, ks *v1alpha1.KnativeServ
 		common.AppendTarget,
 		ingress.AppendTargetIngresses,
 		r.filterDisabledIngresses,
+		r.appendExtension,
```

Comment:
should we name it `appendExtensionManifests` or the like?

We can do that in a different PR, but just wondering, since it's not appending the extension at whole, right ?

Reply:
Ok .